internal/providers/terraform/azure/testdata/notification_hubs_test/notification_hubs_test.tf

```diff
@@ -0,0 +1,55 @@
+provider "azurerm" {
+  skip_provider_registration = true
+  features {}
+}
+
```

Comment:
There should be a `no_usage` case to cover when no ` monthly_additional_pushes` is set in the usage.yml.  I expect it to show something like:
```
└─ Additional Pushes (10-100M)          Monthly cost depends on usage: $10.00 per 1M pushes
```